modules/knative-eventing-CR-system-deployments.adoc

```diff
@@ -31,6 +32,10 @@ metadata:
   namespace: knative-eventing
 spec:
   deployments:
+  - name: FIXME
```

Comment:
You might want to use `eventing-controller` here. How about the Kafka CR?
cc @pierDipi if he wants a better example.

Reply:
Opened an issue for Kafka https://issues.redhat.com/browse/SRVCOM-2206